bundles/org.openhab.binding.http/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -9,6 +9,19 @@
 		<label>HTTP URL Thing</label>
 		<description>Represents a base URL and all associated requests.</description>
 
+		<channels>
+			<channel typeId="requestDateTime" id="lastFailure">
```

Comment:
If these are the first channels, we should name them according to [naming convention](https://www.openhab.org/docs/developer/guidelines.html#naming-convention):
```suggestion
			<channel typeId="request-datetime" id="last-failure">
```